doc_indexer/src/utils/models.py

```diff
@@ -6,6 +6,8 @@
 from gen_ai_hub.proxy.langchain.openai import OpenAIEmbeddings
 from langchain_core.embeddings import Embeddings
 
+# TODO: re-use the model factory parent project
```

Comment:
Is it todo suppose to be completed as part of this PR?

Reply:
it should be part of this [issue](https://github.com/kyma-project/kyma-companion/issues/243). Do you want me to add the issue link to the comment too?

Reply:
Please remove the todo comment from the code then. Having it in the issue is enough.